platform/rpc/client.go

```diff
@@ -91,6 +91,20 @@ func (p *RPCClient) Version() (string, error) {
 	return version, err
 }
 
+// Export is used to get service configuration in platform-specific format
+func (p *RPCClient) Export(ids []flux.ServiceID) (string, error) {
+	var config string
+	err := p.client.Call("RPCServer.Export", ids, &config)
+	if _, ok := err.(rpc.ServerError); !ok && err != nil {
+		return "", platform.FatalError{err}
+	} else if err != nil && err.Error() == "rpc: can't find method RPCServer.Export" {
```

Comment:
I would much rather `Export` formed part of a new protocol, so there weren't special cases like this. (If we change more things, which seems inevitable .. more special cases ...)